pyproject.toml

```diff
@@ -0,0 +1,195 @@
+  {name = "Milo Casagrande" , email = "milo@ubuntu.com"},
+  {name = "Arisu Tachibana" , email = "arisu.tachibana@miraclelinux.com"},
+  {name = "Ed Smith" , email = "ed.smith@collabora.com"},
+  {name = "Heiko Thiery" , email = "heiko.thiery@gmail.com"},
+  {name = "Sjoerd Simons" , email = "sjoerd@collabora.com"},
+  {name = "Alex P" , email = "alexandra.pereira@collabora.com"},
+  {name = "Detlev Casanova" , email = "detlev.casanova@collabora.com"},
+  {name = "Guillaume Tucker" , email = "guillaume@mangoz.org"},
+  {name = "Jerome Brunet" , email = "jbrunet@baylibre.com"},
+  {name = "Nick Desaulniers" , email = "ndesaulniers@google.com"},
+  {name = "Ricardo Ribalda" , email = "ribalda@chromium.org"},
+  {name = "Chase Qi" , email = "chase.qi@linaro.org"},
+  {name = "Florian Bezdeka" , email = "florian.bezdeka@siemens.com"},
+  {name = "Kevin Hilman" , email = "khilman@kernel.org"},
+  {name = "Kevin Hilman" , email = "khilman@linaro.org"},
+  {name = "Lakshmipathi.G" , email = "lakshmipathi.ganapathi@collabora.com"},
+  {name = "Marc Titinger" , email = "mtitinger@baylibre.com"},
+  {name = "Mark Brown" , email = "broonie@sirena.org.uk"},
+  {name = "Matt Hart" , email = "matt@blacklabsystems.com"},
+  {name = "Matt Hart" , email = "matt@mattface.org"},
+  {name = "Mattijs Korpershoek" , email = "mkorpershoek@baylibre.com"},
+  {name = "Milo Casagrande" , email = "milo@opensourcefoundries.com"},
+  {name = "Suram Chandra Sekhar" , email = "suram@nxp.com"},
+  {name = "Suram Chandra Sekhar" , email = "suram@oss.nxp.com"},
+  {name = "Tyler Baker" , email = "tyler@opensourcefoundries.com"},
+  {name = "William K Lin" , email = "lin.wen.kuang@gmail.com"},
+  {name = "alan bennett" , email = "akbennett@hasdell"},
+  {name = "isaiahduncan" , email = "32970507+isaiahduncan@users.noreply.github.com"},
+  {name = "lollivier" , email = "lollivier@baylibre.com"},
+  {name = "Alexandra" , email = "alexandra@arch.localhost"},
+  {name = "Alexandre Belloni" , email = "alexandre.belloni@free-electrons.com"},
+  {name = "Alice Ferrazzi" , email = "alice.ferrazzi@cybertrust.co.jp"},
+  {name = "Ana Guerrero Lopez" , email = "ana@ekaia.org"},
+  {name = "AngeloGioacchino Del Regno" , email = "kholk11@gmail.com"},
+  {name = "Ard Biesheuvel" , email = "ard.biesheuvel@linaro.org"},
+  {name = "Bastian Krause" , email = "bst@pengutronix.de"},
+  {name = "Bjorn Andersson" , email = "bjorn.andersson@linaro.org"},
+  {name = "Denys Fedoryshchenko" , email = "nuclearcat@nuclearcat.com"},
+  {name = "Fathi Boudra" , email = "fathi.boudra@linaro.org"},
+  {name = "Gustavo Padovan" , email = "gustavo.padovan@collabora.com"},
+  {name = "Heidi Fahim" , email = "heidifahim@google.com"},
+  {name = "Jakob Unterwurzacher" , email = "jakob.unterwurzacher@theobroma-systems.com"},
+  {name = "Jan-Simon Möller" , email = "dl9pf@gmx.de"},
+  {name = "Kees Cook" , email = "keescook@chromium.org"},
+  {name = "Lakshmipathi.G" , email = "Lakshmipathi.Ganapathi@Collabora.com"},
```

Comment:
Ditto - email doesn't exist anymore.

It's impressive how many email addresses Laks has used :)